store/sqlstore/license_store.go

```diff
@@ -46,6 +54,9 @@ func (ls SqlLicenseStore) Save(license *model.LicenseRecord) (*model.LicenseReco
 	return &storedLicense, nil
 }
 
+// Get obtains from the database the license with the provided id parameter.
```

Comment:
```suggestion
// Obtains the license with the provided ID parameter from the database. 
```

Reply:
The functions documentation (by design in golang documentation system) must start with the function name, so in this case must start with the `Get` and then the explanation of what `Get` does.

Reply:
The parameter itself is `id` with lowercase, we are referring the parameter name here, so it should match with the casing too.